package.json

```diff
@@ -23,7 +23,8 @@
     "@types/react-redux": "^7.1.25",
     "@types/react-router-dom": "^5.3.3",
     "@types/redux-logger": "^3.0.9",
-    "axios": "^1.2.2",
+    "axios": "^1.2.4",
```

Comment:
Since we are not using axios anymore, we better remove it from the package list.

Reply:
> Since we are not using axios anymore, we better remove it from the package list.

Good idea, I have removed axios :)